mne/io/kit/constants.py

```diff
@@ -164,6 +165,7 @@
 KIT.SYSTEM_YOKOGAWA_2017_01 = 1001  # Kanazawa (until 2017)
 KIT.SYSTEM_YOKOGAWA_2018_01 = 10020  # Kanazawa (since 2018)
 KIT.SYSTEM_YOKOGAWA_2020_08 = 10021  # Kanazawa (since August 2020)
+KIT.SYSTEM_BERLIN = 124
```

Comment:
@UrbanM any idea when the system was installed? I think we try to name these based on that

Reply:
I contacted the manager of the device, I will give you the answer soon.

Reply:
@larsoner I contacted the system administrator. Regarding the date of first installation he said: "Lets set as the installation date 2008 as in this year the system got 128 SQUID channels, before it had only 96. In 2018 the software was upgraded."  
And also his suggestion, if that makes sense:
"My suggestion for naming in constants.py:
`# Known KIT systems`
`KIT.SYSTEM_EAGLE_TECHNOLOGY_PTB = 124 # PTB (since 2008, software upgrade in 2018)`
and
`KIT_SYSNAMES = {`
`KIT.SYSTEM_EAGLE_TECHNOLOGY_PTB: 'Eagle Technology MEG (KIT/Yokogawa style) at PTB (since 2008, software upgrade in 2018)'`
Please pass on this information if it makes sense for you!"